playbooks/common/openshift-master/config.yml

```diff
@@ -180,6 +180,7 @@
     openshift_master_hosts: "{{ groups.oo_masters_to_config }}"
   - role: nickhammond.logrotate
   - role: nuage_master
+    serial: 1
```

Comment:
@vishpat  I'm not sure serial is supported in this context, I believe it is only applicable at the play level.

@abutcher could you follow up on that ^?

@vishpat it might be better to break out the nuage_master/nuage_master role applications to their own play.